vmaas_sync/metrics.go

```diff
@@ -86,11 +87,24 @@ var (
 		Subsystem: "vmaas_sync",
 		Name:      "backend_processes",
 	}, []string{"state"})
+
+	cyndiStats = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
I would use only `cyndiSystemsCnt` to measure systems count with specific properties (e.g. systems with some tags) and `cyndiTagsCnt` for tags counts (e.g. "maxPerSystem", "minPerSystem", cond. "avgPerSystem"). It would be also interesting to have count of unique tags used across all systems.